src/app-config/auth.ts

```diff
@@ -130,6 +134,7 @@ export default class AuthClient {
         payload: { 'client_id': this.config.oauth_client_id },
         headers: { 'HOST': url.hostname },
         rejectUnauthorized: !url.hostname.endsWith('.localhost'),
+        timeout: 30000,
```

Comment:
is this intentionally disconnected from `EXPIRATION_WINDOW_IN_SECONDS` or should this be `EXPIRATION_WINDOW_IN_SECONDS * 100`?

Reply:
Has no relation. This is just an arbitrary timeout for an http call.

Reply:
coo 👍